scanners/trivy/values.yaml

```
@@ -100,6 +100,9 @@ scanner:
   # scanner.tolerations -- Optional tolerations settings that control how the scanner job is scheduled (see: https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/)
   tolerations: []
 
+# kubeauditScope -- Automatically sets up rbac roles for kubeaudit to access the resources it scans. Can be either "cluster" (ClusterRole) or "namespace" (Role)
+kubeauditScope: "cluster"
+
   # -- if set to true the scan job will be suspended after creation. You can then resume the job using `kubectl resume <jobname>` or using a job scheduler like kueue
   suspend: false
 
```

I get the following error when trying to deploy the scanner:
```bash
Error: cannot load values.yaml: error converting YAML to JSON: yaml: line 106: did not find expected key
```
You want .Values.kubeauditScope so it should look like the following

```suggestion
  # -- if set to true the scan job will be suspended after creation. You can then resume the job using `kubectl resume <jobname>` or using a job scheduler like kueue
  suspend: false
  
kubeauditScope: "cluster"

```

---

Recommendation adopted